mappings/net/minecraft/client/render/model/ModelLoader.mapping

```diff
@@ -7,9 +7,22 @@ CLASS net/minecraft/class_1088 net/minecraft/client/render/model/ModelLoader
 		METHOD method_21608 create (Lnet/minecraft/class_2680;Lnet/minecraft/class_1100;Ljava/util/Collection;)Lnet/minecraft/class_1088$class_4455;
 		METHOD method_21609 getValues (Lnet/minecraft/class_2680;Ljava/util/Collection;)Ljava/util/List;
 	FIELD field_17907 spriteAtlasData Lnet/minecraft/class_1059$class_4007;
+	FIELD field_20272 blockColors Lnet/minecraft/class_324;
 	FIELD field_20273 currentModelIndex I
 	FIELD field_20274 stateToModelIndex Lit/unimi/dsi/fastutil/objects/Object2IntMap;
+	FIELD field_20845 SHULKER Lnet/minecraft/class_2960;
+	FIELD field_20846 SHULKER_COLORS Ljava/util/List;
+	FIELD field_20847 BANNER_BASE Lnet/minecraft/class_2960;
+	FIELD field_20848 DESTROY_STAGES Ljava/util/List;
+	FIELD field_21014 OAK_SIGN Lnet/minecraft/class_2960;
+	FIELD field_21015 SPRUCE_SIGN Lnet/minecraft/class_2960;
+	FIELD field_21016 BIRCH_SIGN Lnet/minecraft/class_2960;
+	FIELD field_21017 ACACIA_SIGN Lnet/minecraft/class_2960;
+	FIELD field_21018 JUNGLE_SIGN Lnet/minecraft/class_2960;
+	FIELD field_21019 DARK_OAK_SIGN Lnet/minecraft/class_2960;
 	FIELD field_21020 BLOCK_BREAKING_STAGES Ljava/util/List;
+	FIELD field_21557 SHIELD_BASE Lnet/minecraft/class_2960;
+	FIELD field_21558 NO_PATTERN_SHIELD_BASE Lnet/minecraft/class_2960;
```

Comment:
You're overlapping with my mappings in PR #799

Reply:
can have yours merged first and then I fix conflicts